### ISSUE TITLE
`solana-keygen new` panics if --no-outfile specified

### ISSUE THREAD

**Issue by bbhart** [solana-labs/solana#17860]
#### Problem

Trying to create paper wallet per https://docs.solana.com/wallet-guide/paper-wallet, solana-keygen cli exits abruptly if --no-outfile is specified. 

```
hart_b@Fusion:~$ RUST_BACKTRACE=1 solana-keygen new --no-outfile
thread 'main' panicked at 'Fatal internal error. Please consider filing a bug report at https://github.com/clap-rs/clap/issues', /var/lib/buildkite-agent/.cargo/registry/src/github.com-1ecc6299db9ec823/clap-2.33.3/src/app/usage.rs:475:22
stack backtrace:
   0: rust_begin_unwind
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/std/src/panicking.rs:493:5
   1: core::panicking::panic_fmt
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/panicking.rs:92:14
   2: core::option::expect_failed
             at /rustc/9bc8c42bb2f19e745a63f3445f1ac248fb015e53/library/core/src/option.rs:1321:5
   3: clap::app::usage::get_required_usage_from
   4: clap::app::validator::Validator::missing_required_error
   5: clap::app::validator::Validator::validate_matched_args
   6: clap::app::validator::Validator::validate
   7: clap::app::parser::Parser::get_matches_with
   8: clap::app::parser::Parser::get_matches_with
   9: clap::app::App::get_matches_from_safe_borrow
  10: clap::app::App::get_matches
  11: solana_keygen::main
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.
```

```
hart_b@Fusion:~$ uname -a
Linux Fusion 4.19.128-microsoft-standard #1 SMP Tue Jun 23 12:58:10 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux

hart_b@Fusion:~$ solana-keygen --version
solana-keygen 1.7.1 (src:cd6e1d92; feat:3097519246)
```

**Comment by jaredwolff** [comment]
Same here on OSX. (10.15.7)

**Comment by CriesofCarrots** [comment]
Thank you @bbhart and @jaredwolff ! This will be fixed in v1.7.2 . In the meantime, feel free to use v1.6.11 for keypair generation.
